src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -110,5 +111,26 @@ inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc)
     return arm_compute::DataLayout::UNKNOWN;
 }
 
+/**
+* @brief Return static mutex for lock ComputeLibrary configuration
+* @return static std::mutex
+*/
+static arm_compute::Mutex & getMutexConfig() {
+    static arm_compute::Mutex mtx_config;
+    return mtx_config;
+}
+
+/**
+* @brief run thread-safe configure for ComputeLibrary configuration function
+* @param config ComputeLibrary configuration function
+*/
+inline void lockACLConfiguration(std::function<void(void)> config) {
+    // We get a problem (seg. faults, data race etc) for eltwise operations when we use several configure(...) functions in parallel.
```

Comment:
Please put it into the doxygen part above instead.
"ACL 28.08 does not officially support thread-safe configure() calls.
 For example, calling configure for Eltwise operations from multiple streams leads to a data race and seg fault."

Reply:
@EgorDuplensky corrected it